README.md

```diff
@@ -0,0 +1,10 @@
+# Zincati
+
+[![Build status](https://travis-ci.org/coreos/zincati.svg?branch=master)](https://travis-ci.org/coreos/zincati)
+
+Zincati is the update agent for Fedora CoreOS hosts.
+
+It works as a client for [Cincinnati] and [rpm-ostree], and it takes care of automatic updating/rebooting machines,
```

Comment:
Looks like a trailing comma instead of period here.

Reply:
Arrgh!